hooks/hookexecution/executor.go

```diff
@@ -165,6 +169,11 @@ func (e *hookExecutor) ExecuteProcessedAuctionStage(request *openrtb2.BidRequest
 	e.saveModuleContexts(contexts)
 	e.pushStageOutcome(outcome)
 
+	// remove type information if there is no rejection
+	if reject == nil {
+		return nil
+	}
```

Comment:
I prefer to leave this in because returning a `nil` reject is not the same as returning `nil`. `return reject` contains type information which caused problems during tests. I don't think it would cause problems during normal runtime. I added a comment above to try to explain this situation, but based on your comment that's not a good comment. Perhaps you can recommend a different wording?

Reply:
> `return reject` contains type information which caused problems during tests.

Wow 😮 . I guess I need to get more familiar with the modules package. No, your comment is fine, it totally went over me, sorry.